src/main/java/binnie/core/triggers/TriggerWorking.java

```diff
@@ -1,43 +0,0 @@
-package binnie.core.triggers;
-
-import binnie.core.machines.Machine;
-import binnie.core.machines.power.IProcess;
-
-// TODO unusing class?
-public class TriggerWorking {
```

Comment:
Without implementing some BuildCraft API interfaces?
Need to check it, of course. I'll check during the day.